docs/gatsby-config.js

```diff
@@ -17,17 +17,13 @@ module.exports = {
             'getting-started',
             'configuring',
             'ci-cd',
-            'essentials',
```

Comment:
should we consider adding a redirect to conventions from essentials?

Reply:
we could, but I don't think it's stricly necessary. If we were in GA, I'd say probably

Reply:
Uh oh looks like this was necessary! I'll add momentarily